HPXMLtoOpenStudio/schematron_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'pathname'
+require 'schematron-nokogiri'
+
+file_dir = File.dirname(__FILE__)
+
+# load the schematron xml
+stron_doc = Nokogiri::XML File.open(File.join(file_dir, 'resources', 'EPvalidator.stron'))  # "/path/to/schema.stron"
+# make a schematron object
+stron = SchematronNokogiri::Schema.new stron_doc
+# load the xml document you wish to validate
+xml_doc = Nokogiri::XML File.open(File.join(File.dirname(file_dir), 'workflow', 'sample_files', 'base.xml'))  # "/path/to/xml_document.xml"
+# validate it
+results = stron.validate xml_doc
```

Comment:
I think that basically every rule that's added to the schematron file should have a test that hits it. So, if the schematron file has:
```
<sch:rule context="/hpxml_v3:HPXML/hpxml_v3:Building/hpxml_v3:ProjectStatus">
      <sch:assert test="count(hpxml_v3:EventType) = 1">element 'hpxml_v3:ProjectStatus/hpxml_v3:EventType' is REQUIRED</sch:assert>
</sch:rule>
```
We should have a test where we get a hpxml object from base.xml, remove the ProjectStatus element, validate it against the schematron file, and assert that the schematron error message is generated.